velox/connectors/Connector.h

```diff
@@ -146,6 +146,10 @@ class DataSink {
     return 0;
   }
 
+  virtual int32_t getNumWrittenFiles() const {
```

Comment:
```
s/getNumWrittenFiles/numWrittenFiles/
```
If there is no other connector implementation, can we just throw with VELOX_NYI?

Reply:
The function above is called `getCompletedBytes`, so I will keep the name to stay consistent.

Reply:
Let's remove get prefix for the new method? Thanks!